### ISSUE TITLE
dateien

### ISSUE THREAD

**Issue by joshinils** [philsupertramp/wiki#8]
* darstellung nach dateityp, .`txt`, `.png`, `.stl`, `.py`, `.md`, `.cpp` bei einbindung
* verlauf der änderungen
* eigene seite mit beschreibung pro datei
* neue datei hochladen, kommentarmöglichkeit dazu
* datei ersetzen mit neuer version, kommentarmöglichkeit dazu
* welche seiten nutzen diese datei? auflistung
* download der versionen
* anzeigen der datei als raw-format, oder als interpretiertes format mittels "viewer" (syntax highlighting? zoom bei bildern?)

> PZ:
Ja und nein.. Problem ist, webspace kostet. Die Idee war auch eigentlich den Kontent sichtbar zu haben, ansonsten könnten wir ja gleich nen FTP server nutzen.
Meine Kompromisslösung wären Code blocks, die man seperat angeben kann, in der art von [https://gist.github.com/](gists). Binärdateien sind aber unmöglich.

**Comment by joshinils** [comment]
also muss das einbinden von bildern irgendwie möglich sein, und relativ einfach gestaltbar im source der seite. wo die dann liegen, naja wohl nicht beim wiki...

**Comment by joshinils** [comment]
code blocks möchte ich ja lieber mit den templates machen, als explizit als datei, dann sind die teil des source der seite, nicht eingebunden

**Comment by philsupertramp** [comment]
> also muss das einbinden von bildern irgendwie möglich sein, und relativ einfach gestaltbar im source der seite. wo die dann liegen, naja wohl nicht beim wiki... 

Ja, du kannst es extern hochladen und Markdown für das anzeigen nutzen, da kannst du Bilder anzeigen wie hier

![IMG-20210206-WA0000.jpg](https://user-images.githubusercontent.com/9550040/107608956-c80c8280-6c3d-11eb-9041-20211269a524.jpg)
